libdnf/CMakeLists.txt

```diff
@@ -44,6 +44,10 @@ pkg_check_modules(LIBFMT REQUIRED fmt)
 list(APPEND LIBDNF_PC_REQUIRES "${LIBFMT_MODULE_NAME}")
 target_link_libraries(libdnf ${LIBFMT_LIBRARIES})
 
+pkg_check_modules(LIBGPGME REQUIRED gpgme)
+list(APPEND LIBDNF_PC_REQUIRES "${LIBGPGME_MODULE_NAME}")
+target_link_libraries(libdnf ${LIBGPGME_LIBRARIES})
+
```

Comment:
The gpgme check is already present on line 84 of this file :slightly_smiling_face: